lib/p2p/Pool.ts

```diff
@@ -466,7 +466,7 @@ class Pool extends EventEmitter {
       case PacketType.OrderInvalidation: {
         const order = (packet as packets.OrderInvalidationPacket).body!;
         this.logger.verbose(`canceled order from ${peer.nodePubKey}: ${JSON.stringify(order)}`);
-        this.emit('packet.orderInvalidation', order);
+        this.emit('packet.orderInvalidation', { ...order, peerPubKey: peer.nodePubKey } as OrderInvalidation);
```

Comment:
I also have a performance concern here, where with the rest operator again we're creating a new object which is a minor performance hit but one that we'd have to do many times. I think it would be just as clear if we used two parameters, the `OrderPortion` and the `peer.nodePubKey` as a separate string. We just turn around and pass them in as separate parameters to `removePeerOrder` anyway, so we're not gaining much by having these combing into a single object as I see it.